serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc

```diff
@@ -6,11 +6,11 @@
 // links
 :quarkus_issue_url: https://github.com/quarkusio/quarkus/issues/27457
 
-The Kubernetes service discovery allows you to have a static URI, defining a Kubernetes resource, which is used to perform HTTP requests. The Kubernetes resource defined in the URI is queried in the current Kubernetes cluster and translated in a valid URL. 
+The Kubernetes service discovery allows you to describe the Kubernetes resource you want to perform an HTTP request using a custom URI. Under the hood, it will discover the network endpoint (URL) to where to make the request.
```

Comment:
```suggestion
The Kubernetes service discovery allows you to describe the Kubernetes resource you want to perform an HTTP request using a custom URI. Under the hood, it will discover the network endpoint (URL) to where to make the request.
```